src/fromager/__main__.py

```diff
@@ -68,6 +68,13 @@
     type=clickext.ClickPath(),
     help="location to manage working files, including builds",
 )
+@click.option(
+    "-t",
```

Comment:
`-t` is already being used by `--word-dir`. Use another shorthand like `-l` maybe?

Reply:
Do we need a separate log dir? We could store the logs in `ctx.work_dir / "logs"`.

Reply:
Did Doug say why he wanted a different directory? I don't see a comment about the log directory from him.

The work directory also contains a bunch of temporary stuff like unpacked sdists and wheels. It makes sense to have a separate directory with state that we want to archive and retain for future use. What do you think about a more general name (state, output, ...)? We could move the graph, build-order, and constraints file into the directory and the new logs in a "logs" subdirectory.

Reply:
I guess @shubhbapna is working on a similar feature for archiving `sdists` and I remember Doug mentioning this so that we both are on same track. Also, he mentioned that this format will help us use the upload plugin which we have in builder to upload this dir.

I can move it to a sub-directory inside` work-dir` if that's a better approach :)

Reply:
I have updated the code to make `logs` as sub-directory of `work-dir`